src/modules/70-pipeline/components/PipelineStudio/PipelineVariables/PipelineVariables.tsx

```diff
@@ -146,23 +127,117 @@ export function PipelineVariablesWithRef(
     closeDrawer()
   }
 
+  return (
+    <div className={css.pipelineVariables}>
+      {error ? (
+        <PageError message={(error?.data as Error)?.message || error?.message} />
+      ) : (
+        <div className={css.content}>
+          <VariablesHeader applyChanges={applyChanges} discardChanges={discardChanges} />
+          <div className={css.variableList} ref={pipelineVariablesRef as any}>
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232314)
```suggestion
          <div className={css.variableList} ref={pipelineVariablesRef as unknown}>
```